app/components/base/Hero.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+import { breakDown } from "../breakpoints";
+import hero from "../../../assets/images/content-images/hero.png";
+
+const StyledHero = styled.div`
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: right;
+  position: relative;
+  width: 100%;
+
+  @media (${breakDown("sm")}) {
+    display: none;
+  }
+
+  @media (${breakDown("xs")}) {
```

Comment:
We can probably get rid of this `@media` block since the breakdown for `"sm"` will account for "xs" as well.